api/server/graph.cc

```diff
@@ -152,6 +152,8 @@ bool Graph::start(int argc, char **argv) {
     return true;
 }
 
+#define POLLER_CHECK(c) ((c) & 1023)
+#define FIREWALL_GC(c) ((c) == 50000)
```

Comment:
I Would have use 100000 for this value, as cnt was reset at 100000 before this patch  Pg::firewall_gc was call each 100000 turn.